### PR TITLE
Add Laravel 11 support

### PR DESCRIPTION
**Pull request by yurichechulin**
### Added

- Laravel 11 support

### CODE REVIEW

File: sdk/php/phpunit.xml.dist.bak
Comment (eldario) [inline sdk/php/phpunit.xml.dist.bak:1]: What is this file for?

File: sdk/php/src/VehicleLogotypes.php [hunk 1]
```diff
@@ -15,8 +15,9 @@ final class VehicleLogotypes extends \Illuminate\Support\Collection
      */
     public static function create(): self
     {
-        return new self(
-            \json_decode((string) \file_get_contents(__DIR__ . '/../../../src/vehicle-logotypes.json'), true, 512, JSON_THROW_ON_ERROR)
-        );
+        /** @var array<string, mixed> $data */
+        $data = \json_decode((string) \file_get_contents(__DIR__ . '/../../../src/vehicle-logotypes.json'), true, 512, JSON_THROW_ON_ERROR);
```
Comment (eldario) [inline sdk/php/src/VehicleLogotypes.php:19]: This won't help?
```php
        return new self(
            (array) \json_decode((string) \file_get_contents(__DIR__ . '/../../../src/vehicle-logotypes.json'), true, 512, JSON_THROW_ON_ERROR)
        );
```

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,20 @@
-FROM php:8.0-alpine
+FROM php:8.1-alpine
 
-ENV COMPOSER_HOME="/tmp/composer"
+ENV \
+    COMPOSER_ALLOW_SUPERUSER="1" \
+    COMPOSER_HOME="/tmp/composer"
 
-COPY --from=composer:2.0.7 /usr/bin/composer /usr/bin/composer
+COPY --from=composer:2.6.6 /usr/bin/composer /usr/bin/composer
```
Comment (eldario) [inline Dockerfile:7]: There is a newer version `v2.7`

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -18,8 +18,8 @@ jobs: # Docs: <https://git.io/JvxXE>
     strategy:
       fail-fast: false
       matrix:
-        setup: [basic, lowest]
-        php: ['7.3', '7.4', '8.0']
+        setup: [ 'basic', 'lowest' ]
```
Comment (eldario) [inline .github/workflows/tests.yml:21]: Don't forget to remove the argument `--no-suggest`

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -57,7 +57,7 @@ jobs: # Docs: <https://git.io/JvxXE>
           XDEBUG_MODE: coverage
         run: composer test-cover
 
-      - uses: codecov/codecov-action@v1 # Docs: <https://github.com/codecov/codecov-action>
+      - uses: codecov/codecov-action@v3 # Docs: <https://github.com/codecov/codecov-action>
```
Comment (eldario) [inline .github/workflows/tests.yml:60]: Try to upgrade `v4`

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -18,8 +18,8 @@ jobs: # Docs: <https://git.io/JvxXE>
     strategy:
       fail-fast: false
       matrix:
-        setup: [basic, lowest]
-        php: ['7.3', '7.4', '8.0']
+        setup: [ 'basic', 'lowest' ]
+        php: [ '8.0','8.1','8.2','8.3' ]
     steps:
       - name: Check out code
         uses: actions/checkout@v2
```
Comment (eldario) [inline .github/workflows/tests.yml:25]: `v4`

File: .github/workflows/tests.yml [hunk 6]
```diff
@@ -32,12 +32,12 @@ jobs: # Docs: <https://git.io/JvxXE>
 
       - name: Get Composer Cache Directory # Docs: <https://git.io/JfAKn#php---composer>
         id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+        run: echo "output_dir=dir::$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
 
       - name: Cache dependencies # Docs: <https://git.io/JfAKn#php---composer>
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```
Comment (eldario) [inline .github/workflows/tests.yml:38]: `v4`

File: Dockerfile [hunk 7]
```diff
@@ -1,17 +1,20 @@
-FROM php:8.0-alpine
+FROM php:8.1-alpine
 
-ENV COMPOSER_HOME="/tmp/composer"
+ENV \
+    COMPOSER_ALLOW_SUPERUSER="1" \
```
Comment (eldario) [inline Dockerfile:4]: What is this for?

File: composer.json [hunk 8]
```diff
@@ -15,14 +15,14 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
-        "illuminate/support": "~6.0 || ~7.0 || ~8.0 || ~9.0",
+        "php": "^8.0.2",
```
Comment (eldario) [inline composer.json:18]: Why was this version chosen?

**Review by eldario — CHANGES_REQUESTED** [review]

File: sdk/php/phpunit.xml.dist.bak
Comment (yurichechulin) [reply sdk/php/phpunit.xml.dist.bak:1]: it was a backup for deprecated phpunit configuration. Removed this file, thx

File: .github/workflows/tests.yml (hunk 3 above)
Comment (yurichechulin) [reply .github/workflows/tests.yml:21]: Removed

File: .github/workflows/tests.yml (hunk 5 above)
Comment (yurichechulin) [reply .github/workflows/tests.yml]: changed to v4

File: .github/workflows/tests.yml (hunk 6 above)
Comment (yurichechulin) [reply .github/workflows/tests.yml]: changed to v4

File: .github/workflows/tests.yml (hunk 4 above)
Comment (yurichechulin) [reply .github/workflows/tests.yml]: changed to v4

File: Dockerfile (hunk 7 above)
Comment (yurichechulin) [reply Dockerfile]: removed  `COMPOSER_ALLOW_SUPERUSER="1" \`

File: Dockerfile (hunk 2 above)
Comment (yurichechulin) [reply Dockerfile]: Now used v2.7 version

File: composer.json (hunk 8 above)
Comment (yurichechulin) [reply composer.json]: Changed version to `^8.0`

File: sdk/php/src/VehicleLogotypes.php (hunk 1 above)
Comment (yurichechulin) [reply sdk/php/src/VehicleLogotypes.php]: it helps, thx

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
 
+## Unreleased
+
+### Added
+
+- Support Laravel versions `10.x` and `11.x`
+
+### Changed
+
+- Minimal `PHP` version now is `^8.0`
```
Comment (eldario) [inline CHANGELOG.md:15]:
```suggestion
- Minimal require PHP version now is `8.0`
- Minimal Laravel version now is `9.0`
- Version of `composer` in docker container updated up to `2.7`
```

**Review by eldario — APPROVED** [review]

**Comment by eldario** [comment]
@yurichechulin 

Thanx for ur PR!

**Pull request merged**